### PR TITLE
HUM-529 Gating job for rpc-hummingbird

### PR DESCRIPTION
**Pull request by andymcc**
Issue: [HUM-529](https://rpc-openstack.atlassian.net/browse/HUM-529)

### CODE REVIEW

**Comment by andymcc** [comment]
recheck_cit_xenial_functional_test

**Comment by andymcc** [comment]
recheck_cit_re_unit_tests

**Comment by andymcc** [comment]
recheck_cit_xenial_lint_test

**Comment by git-harry** [comment]
recheck_unit_tests

**Comment by andymcc** [comment]
@mattt416 i think they do (same way as ceph really)

**Comment by creiht** [comment]
Yeah we are deploying rpc-maas for hummingbird the same way ceph does.

**Comment by git-harry** [comment]
Looking at a recent Ceph build [1] I appears that only local verification is done so I'm fairly certain that means the credentials aren't required. @mattt416 do you know if that's the case?

[1] https://rpc.jenkins.cit.rackspace.net/job/PM_rpc-ceph-master-xenial-functional-test/21/consoleFull

**Comment by andymcc** [comment]
@git-harry if that's the case I'll have to fix that up, I had changed it to "use_api" which I thought would mean it wasn't using local anymore, but I'll dig into that. Regardless, the plan is to not do local only checks.

File: rpc_jobs/rpc_hummingbird.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+    branches:
+      - "master"
+
+    image:
+      - xenial:
+          FLAVOR: "general1-8"
+          IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
+
+    # rpc-ceph ignores that setting for now
```
Comment (git-harry) [inline rpc_jobs/rpc_hummingbird.yml]: rpc-hummingbird?

File: rpc_jobs/rpc_hummingbird.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    scenario:
+      - "functional"
+
+    # rpc-ceph ignores that setting for now
```
Comment (git-harry) [inline rpc_jobs/rpc_hummingbird.yml]: rpc-hummingbird?

File: rpc_jobs/rpc_hummingbird.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+    image:
+      - xenial:
+          FLAVOR: "general1-8"
+          IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
+
+    scenario:
+      - "functional"
+
+    # rpc-ceph ignores that setting for now
```
Comment (git-harry) [inline rpc_jobs/rpc_hummingbird.yml]: rpc-hummingbird?

**Comment by mattt416** [comment]
I'm pretty sure the creds were required when we moved rpc-ceph over to the standard template, because the jobs were initially failing until I added the MaaS creds back in.

**Comment by andymcc** [comment]
@git-harry updated - good spot. should've used sed!

**Comment by mattt416** [comment]
recheck_xenial_functional_test

**Pull request merged**